MeetupPlatformApi/Features/Authentication/GetUser/UserInfoDto.cs

```diff
@@ -17,4 +17,7 @@ public class UserInfoDto
     /// <example>Inan1965</example>
     [Required]
     public string Username { get; set; }
+
+    [Required]
```

Comment:
I may be wrong, but it seems to me that you can set a comment here for Role prop